src/main/java/chess/domain/piece/PieceType.java

```diff
@@ -1,17 +1,41 @@
 package chess.domain.piece;
 
+import chess.domain.piece.blank.Blank;
+import chess.domain.piece.nonsliding.King;
+import chess.domain.piece.nonsliding.Knight;
+import chess.domain.piece.pawn.BlackFirstPawn;
+import chess.domain.piece.pawn.BlackPawn;
+import chess.domain.piece.pawn.WhiteFirstPawn;
+import chess.domain.piece.pawn.WhitePawn;
+import chess.domain.piece.sliding.Bishop;
+import chess.domain.piece.sliding.Queen;
+import chess.domain.piece.sliding.Rook;
+import java.util.function.Supplier;
+
 public enum PieceType {
-    WHITE_PAWN,
-    WHITE_KNIGHT,
-    WHITE_BISHOP,
-    WHITE_ROOK,
-    WHITE_QUEEN,
-    WHITE_KING,
-    BLACK_PAWN,
-    BLACK_KNIGHT,
-    BLACK_BISHOP,
-    BLACK_ROOK,
-    BLACK_QUEEN,
-    BLACK_KING,
-    BLANK;
+    WHITE_FIRST_PAWN(WhiteFirstPawn::new),
+    WHITE_PAWN(WhitePawn::new),
+    WHITE_KNIGHT(Knight::createWhiteKnight),
```

Comment:
PIeceType + PieceDto 덕분에 도메인에 큰 변화없이 저장과 불러오기가 가능했네요 👍 

```suggestion
    WHITE_KNIGHT(() -> new Knight(Color.WHITE)),
```
어떤 건 생성자를 쓰고 어떤 건 정적 메서드를 쓰는 게 어색하네요.

Reply:
앗 이런 방법이 있었네요...😅
감사합니다! 👍👍👍